manifests/init.pp

```diff
@@ -36,15 +36,15 @@
 #    environment variable.
 #
 class java (
-  String $distribution                                              = 'jdk',
-  Pattern[/present|installed|latest|^[.+_0-9a-zA-Z:~-]+$/] $version = 'present',
-  Optional[String] $package                                         = undef,
-  Optional[Array] $package_options                                  = undef,
-  Optional[String] $java_alternative                                = undef,
-  Optional[String] $java_alternative_path                           = undef,
-  Optional[String] $java_home                                       = undef
+  String                                                    $distribution           = 'jdk',
+  Pattern[/present|installed|latest|^[.+_0-9a-zA-Z:~-]+$/]  $version                = 'present',
+  Optional[String]                                          $package                = undef,
+  Optional[Array]                                           $package_options        = undef,
+  Optional[String]                                          $java_alternative       = undef,
+  Optional[String]                                          $java_alternative_path  = undef,
+  Optional[String]                                          $java_home              = undef
 ) {
-  include ::java::params
+  contain java::params
```

Comment:
Not sure why `contain` here instead of `include` (considering `java::params` doesn't declare any resources and only sets variables).